### ISSUE TITLE
Using slqa_session in Meta class declaration.

### ISSUE THREAD

**Issue by fnalonso** [marshmallow-code/flask-marshmallow#180]
Hi,

I am implementing a simple API using flask-marshmallow and the error AttributeError: 'DummySession' object has no attribute 'query' happened, after navigating the source code I found some comments about adding the `sqla_session` inside the Meta class when using the attribute `load_instance = True` and then everything started to work fine. Why this is needed ? I am creating my app instance as bellow:
```python
from models.db import db #flask_sqlalchemy.SQLAlchemy instance
from schemas.ma import ma #flask_marshmallow.Marshmallow instance
app = Flask(__name__)
# models imported
# routes added

db.init_app(app)
ma.init_app(app)
```
Inside the init_app from ma the session is set from app.extensions['sqlalchemy'].db and this is confusing me because I thought this was a global session used for every schema created. This only happen when using the schema.load() method. There is any reason for this behavior ?

Thank you!!

PS: Sorry if this is not the place to put this question.

**Comment by sloria** [comment]
Thanks for reporting. I've been meaning to fix this but lost track of it... https://github.com/marshmallow-code/flask-marshmallow/pull/181 should address this.

**Comment by sloria** [comment]
Fixed in 0.12.0

**Comment by fnalonso** [comment]
Thank you!!
